res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -7,6 +7,8 @@
   <project_license>GPL-2.0</project_license>
   <name>Mixxx DJ Software</name>
   <summary>Everything you need to perform live DJ mixes</summary>
+  <content_rating type="oars-1.1" />
+  <release>2.3.1</release>
```

Comment:
Is this correct? Release tags must be encapsulated within a `<releases>` element.

I am not keen on maintaining yet another literal version constant in the codebase. Do we have other options?

Reply:
We could autogenerate it when tagging.

Reply:
You're right, this is incorrect.